mercury_ide/code/mercury.js

```diff
@@ -185,6 +229,13 @@ const handlers = {
 	'threeFibonacci' : (...v) => {
 		return Algo.threeFibonacci(...v);
 	},
+	// Per Nørgards Inifity series
+	'infinitySeries' : (...v) => {
+		return Algo.infinitySeries(...v);
+	},
+	'infSeries' : (...v) => {
+		return functionMap.infinitySeries(...v);
```

Comment:
All the places where it says `functionMap.` will probably not work, because in this file the object containing all the functions is called `handlers`. So this should either be `handlers.infinitySeries`, or just replace for the same function as under `infinitySeries`, so `Algo.`

Reply:
Thanks. I added a commit that removes the `functionMap` syntax. (I had originally copied this from the `totalSerialismIR.js` code at `mercury-playground` and didn't realize it wouldn't work here.)

For the Markov functions, I removed the alias functions completely - because the only solution I knew would be to copy the full code block from the main Markov functions. This didn't seem good. There may be a better way or maybe it is OK in this case to duplicate the code.